.github/workflows/check.yaml

```diff
@@ -0,0 +1,49 @@
+# Copyright 2022 The Kubevirt Authors.
```

Comment:
Not related to this PR:

We need to change the global lic of this repo as well (and the gitignore)
both those files were created with some template but not updated yet

Reply:
Please keep it `Copyright 2022.` like the other files have.
Once will will update the template I guess it will be `Copyright 2022 The KubeSecondaryDNS Authors.`

Reply:
done

Reply:
Note that some files in this repo have
"# Copyright 2018-2022 Red Hat, Inc."
so maybe that we have all with the same license

and that imo the main LICNESE also need to have the who / year